draft-blind-bbs-signatures.md

```diff
@@ -74,6 +68,16 @@ s'
 blind\_signature
 : The blind digital signature output.
 
+## Notation
+
+Terms specific to pairing-friendly elliptic curves that are relevant to this document are restated in BBS Signature Scheme [TODO add reference], originally defined in [@!I-D.irtf-cfrg-pairing-friendly-curves]
```

Comment:
Can we replace the TODO and add a reference to the core draft using this link (https://identity.foundation/bbs-signature/draft-bbs-signatures.html) using the reference syntax shown at the bottom here https://raw.githubusercontent.com/decentralized-identity/bbs-signature/main/draft-bbs-signatures.md

Reply:
Ok, I am going to commit it